frontend/pulse3d/src/utils/generic.js

```diff
@@ -298,6 +298,18 @@ const formatAdvancedAnalysisJob = (job) => {
   }
 };
 
+const formatNotificationMessage = (msg) => {
+  const { id, created_at, viewed_at, subject, body } = msg;
+
+  return {
+    id,
+    createdAt: created_at,
+    viewed: !!viewed_at,
```

Comment:
Is this to convert to a bool?

Reply:
yes, it was suggested by the IDE and it seems to work